regulations/static/regulations/css/less/typography.less

```diff
@@ -248,17 +248,21 @@ a:visited {
 .definition:visited {
     color: @black;
     text-decoration: none;
-    border-bottom: 1px solid #d0d0ce;
+    border-bottom: 1px solid #D1D3D5
```

Comment:
Good idea! L251 is still splitting the semicolon into another line, but it's good to go after that.